src/components/Realtime.tsx

```diff
@@ -0,0 +1,416 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import { useState } from 'react';
+import theme from '@src/styles/tokens/index';
+
+import product from '@src/assets/mock/itemList_mock';
+
+const Realtime = () => {
+    const [selectedTarget, setSelectedTarget] = useState('ALL');
+    const [selectedSort, setSelectedSort] = useState('wanted');
+
+
+    const [expanded, setExpanded] = useState(false);
+    const displayedCount = expanded ? 21 : 6;
+
+    const targets = [
+        { key: 'ALL', label: '전체', icon: 'ALL' },
+        { key: 'FEMALE', label: '여성이', icon: '👩🏻' },
+        { key: 'MALE', label: '남성이', icon: '👨🏻' },
+        { key: 'TEEN', label: '청소년이', icon: '👦🏻' },
+    ];
+
+    const sortOptions = [
+        { key: 'wanted', label: '받고 싶어한' },
+        { key: 'gifted', label: '많이 선물한' },
+        { key: 'wished', label: '위시로 받은' },
+    ];
+
+    const targetButton = css`
+    width: 3.625rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 4px;
+    background: none;
+    border: none;
+    cursor: pointer;
+  `;
+
+    const buttonDivBase = css`
+    width: 2.75rem;
+    height: 2.75rem;
+    border-radius: 1rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 0.875rem;
+    font-weight: 700;
+    line-height: 1.1875rem;
+    transition: background-color 200ms;
+  `;
+
+    const buttonPBase = css`
+    font-size: 0.8rem;
+    line-height: 1.1875rem;
+    margin: 0px;
+    text-align: left;
+  `;
+
+    const getButtonDivStyle = (key: string) =>
+        css`
+      ${buttonDivBase};
+      color: ${key === selectedTarget ? 'white' : theme.colors.blue400};
+      background-color: ${key === selectedTarget
+                ? theme.colors.blue700
+                : theme.colors.blue100};
+    `;
+
+    const getButtonPStyle = (key: string) =>
+        css`
+      ${buttonPBase};
+      font-weight: ${key === selectedTarget ? 700 : 400};
+      color: ${key === selectedTarget
+                ? theme.colors.blue700
+                : theme.colors.gray700};
+    `;
+
+    const div1Style = css`height: 24px;`;
+
+    const sectionFighting = css`
+    padding: 0px 16px;
+  `;
+
+    const divFighting = css`
+    width: 100%;
+    border-radius: 1rem;
+    background-color: ${theme.colors.yellow600};
+    padding: 16px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+  `;
+
+    const pFightingUp = css`
+    font-size: 0.75rem;
+    font-weight: 400;
+    line-height: 1rem;
+    color: ${theme.colors.gray700};
+    margin: 0px;
+    text-align: left;
+  `;
+
+    const pFightingDown = css`
+    font-size: 0.875rem;
+    font-weight: 700;
+    line-height: 1.1875rem;
+    color: ${theme.colors.textDefault};
+    margin: 0px;
+    text-align: left;
+  `;
+
+    const div2Style = css`height: 40px;`;
```

Comment:
변수명으로 `div1Style`, `div2Style`... 이런 식으로 작성한 것들이 많이 보이는데요. 아시겠지만 이런 네이밍은 유지보수 관점에서 별로 좋지 않아요.
다른 변수들은 잘 작성하신 것을 보면 아무래도 어떻게 써야하는지 헷갈리셨던 것 같아요. 저라면 아래와 같이 작성했을 것 같습니다.
```suggestion
    const spacer40 = css`height: 40px;`;
```